@here/harp-omv-datasource/lib/OmvDataFilter.ts

```diff
@@ -847,6 +707,24 @@ export class OmvGenericFeatureModifier implements OmvFeatureModifier {
         return false;
     }
 
+    protected static matchAttribute(
+        layerName: string,
+        env: MapEnv,
+        items: OmvFilterDescription[]
+    ): boolean {
+        for (const item of items) {
+            if (item.featureAttribute !== undefined) {
+                if (!OmvFilterString.matchString(layerName, item.layerName)) {
```

Comment:
minor, you could consider moving this into the if condition below, i.e. if matchstring && lookup... { return true; }